### DISCUSSION TITLE
Changing patterns when playing live

### DISCUSSION THREAD

**Discussion by Robst01** [hydrogen-music/hydrogen#1661] [Hydrogen on Linux]
Hello. I can't find information on this. How do i change the pattern when playing live?
I tried to setup action "SELECT_NEXT_PATTERN" on an event but it didn't work. Ideally i would like to select a specific pattern with a midi device or keyboard shortcut.

**Comment by theGreatWhiteShark** [comment]
With `SELECT_NEXT_PATTERN` you can switch to a pattern specified in the left-most spin box. To switch to an arbitrary pattern determined by an incoming CC value - e.g. using a control knob on your MIDI keyboard - , you can use `SELECT_NEXT_PATTERN_CC_ABSOLUTE`. 

You can find the full list of available commands here http://hydrogen-music.org/documentation/manual/manual_en.html#chpt.midi.actions
